scripts/us_epa/superfund/sites/tar_creek/process_report2020.py

```diff
@@ -0,0 +1,208 @@
+import os
+import sys
+import tabula
+import pandas as pd
+
+from absl import app, flags
+
+# Allows the following module imports to work when running as a script
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH,
+                             '../../../../'))  # for superfund_vars
+from us_epa.util.superfund_vars import _TAR_CREEK_WELL_MAP
+
+
+def get_table_data_from_pdf(input_path: str, page_str: str) -> list:
+    try:
+        return tabula.read_pdf(input_path,
+                               stream=True,
+                               guess=True,
+                               pages=page_str)
+    except:
+        print(
+            f"ERROR: An error was encountered while extracting tables in {input_path} for {page_str}.\n Please check if the page numbers and files are correct. Also, please note table in images are not extracted."
+        )
+        exit()
+
+
+# python3 process_report2020.py --input_path=<path to the PDF on local computer> --pages=<83-89|83,..,89|83>
+
+
+def process_2020_report(input_path: str,
+                        output_path: str,
+                        page_range: str,
+                        skip_count: int = 5) -> list:
+    """
+    Processes the extracted tabular data with corrections to the data and data cleanup
+    """
+    ## Create output directory if not present
+    if not os.path.exists(output_path):
+        os.makedirs(output_path)
+
+    input_path = "./data/tar_creek_reports/2020_TarCreek.pdf"  #"https://semspub.epa.gov/work/06/100021610.pdf"
```

Comment:
delete comment?

Reply:
Yes, thanks for catching this .. this was from the earlier version of the script . I have cleaned this up now.